packages/react/test/specs/components/Dropdown/Dropdown-test.tsx

```diff
@@ -1502,21 +1428,13 @@ describe('Dropdown', () => {
   })
 
   describe('renderSelectedItem', () => {
-    it('calls renderSelectedItem in multiple selection', () => {
-      const renderSelectedItem = jest.fn((selectedItem, props) => null)
-      const wrapper = mountWithProvider(
-        <Dropdown multiple items={items} renderSelectedItem={renderSelectedItem} />,
-      )
-      const triggerButton = getTriggerButtonWrapper(wrapper)
+    it('is called in multiple selection', () => {
+      const renderSelectedItem = jest.fn()
+      const value = [items[0], items[1]]
 
-      triggerButton.simulate('click')
-      const firstItem = getItemAtIndexWrapper(wrapper)
-      firstItem.simulate('click', { nativeEvent: { stopImmediatePropagation: _.noop } })
-      triggerButton.simulate('click')
-      const secondItem = getItemAtIndexWrapper(wrapper, 1)
-      secondItem.simulate('click', { nativeEvent: { stopImmediatePropagation: _.noop } })
+      renderDropdown({ renderSelectedItem, multiple: true, value })
 
-      expect(renderSelectedItem).toBeCalled()
+      expect(renderSelectedItem).toBeCalledTimes(value.length)
```

Comment:
```suggestion
      expect(renderSelectedItem).toBeCalledTimes(2)
```

nit: I prefer obvious values